diff --git a/contracts/LiquidationsPeripheral.vy b/contracts/LiquidationsPeripheral.vy
index 9a62d7c..f490719 100644
--- a/contracts/LiquidationsPeripheral.vy
+++ b/contracts/LiquidationsPeripheral.vy
@@ -258,6 +258,7 @@ def _computeLoanInterestAmount(principal: uint256, interest: uint256, duration:
 @view
 @internal
 def _getNFTXVaultAddrFromCollateralAddr(_collateralAddress: address) -> address:
+    if 1 > 0: return empty(address)
     if self.nftxVaultFactoryAddress == empty(address):
         return empty(address)
 
